coala

```diff
@@ -15,8 +15,8 @@
 
 import sys
```

Comment:
PEP8Bear is going to apply a patch here which will break the builds again. So we need to move `sys` inside the ignore comment.

Reply:
Okay, changes made.